dbytecoderc/Questioner
# #163415664 validate user input

#### What does this PR do?
> This PR makes sure the user does not submit an inappropriate input

#### Description of Task to be completed?
>Added validation checks using `validatorjs` to validate input and check for appropriate inputs to make sure they are fit to be consumed by the controller

#### How should this be manually tested?
> - Clone the repo and run `npm install` to install dependencies.
> - Download and install postgresql from this [link](https://www.postgresql.org/)
> - Setup postgreql using this [guide](https://www.postgresql.org/docs/10/runtime.html)
> - run `npm migrate` to create the tables
> - run `npm start:dev` to start server
> - check the `server/test/testData` folder route for the input formats and proceed to testing with postman
> - alternatively run `npm run test` to test the endpoints

#### What are the relevant pivotal tracker stories?
> - `#163415664`
